src/box/tuple.h

```diff
@@ -320,9 +320,27 @@ box_tuple_next(box_tuple_iterator_t *it);
 box_tuple_t *
 box_tuple_new(box_tuple_format_t *format, const char *data, const char *end);
 
+/**
+ * Update tuple data in a space.
+ * @param tuple  Tuple to update.
+ * @param expr
+ * @param expr_end
+ *
+ * @retval  NULL The tuple was not updated.
+ * @retval  box_tuple_t The tuple successfully updated in a space.
+ */
 box_tuple_t *
 box_tuple_update(box_tuple_t *tuple, const char *expr, const char *expr_end);
 
+/**
+ * Upsert tuple data to a space.
+ * @param tuple  Tuple to upsert.
+ * @param expr
+ * @param expr_end
+ *
+ * @retval  NULL The tuple was not upserted.
+ * @retval  box_tuple_t The tuple successfully upserted to a space.
+ */
```

Comment:
Same comments as for `box_tuple_update()`, but also I wouldn't say upsert/upserted. It is update or insert, right? We can just say so.

Reply:
Updated, thanks

Reply:
```diff
--- a/src/box/tuple.h
+++ b/src/box/tuple.h
@@ -334,12 +334,12 @@ box_tuple_update(box_tuple_t *tuple, const char *expr, const char *expr_end);
 
 /**
  * Upsert tuple data to a space.
- * @param tuple  Tuple to upsert.
+ * @param tuple  Tuple to update or insert.
  * @param expr
  * @param expr_end
  *
- * @retval  NULL The tuple was not upserted.
- * @retval  box_tuple_t The tuple successfully upserted to a space.
+ * @retval  NULL The tuple was not updated or inserted.
+ * @retval  box_tuple_t The tuple successfully updated or inserted to a space.
  */
 box_tuple_t *
 box_tuple_upsert(box_tuple_t *tuple, const char *expr, const char *expr_end);
```